posters/merge_concatenate_poster_notes.txt

```diff
@@ -0,0 +1,5 @@
+I created both the Merge and Concatenate posters using Inkscape. I have included the svg files used to make the poster in case anyone wants to edit it or base a new poster on it. Everything in the document is based on Inkscape's 3D boxes tool with Angles X, Y and Z set to (154.759, 90.000, 25.241). These angles can be adjusted but I'm not able to get a consistent result from this and it never really behaved the way I expected. 
```

Comment:
Please rename this file to `readme.txt` and make sure that lines are not longer than 80 characters in length.